app/views/practices/form/contact.html.erb

```diff
@@ -3,17 +3,22 @@
 
 <% provide :head_tags do %>
   <%= javascript_include_tag 'practice_editor_utilities', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_tag 'data-turbolinks-track': 'reload' do %>
+    <%= render partial: 'practices/form/session_monitor', formats: [:js] %>
+  <% end %>
 <% end %>
 
 <%= render 'practice_editor_sidenav' %>
 <div class="grid-container position-relative">
   <div class="grid-row grid-gap practice-editor-min-height">
     <div class="desktop:grid-col-3 grid-col-auto z-bottom">&nbsp;</div>
-
     <div id="contact" class="desktop:grid-col-9 grid-col-12 p0">
       <%= render partial: "shared/messages", locals: {small_text: false} %>
       <section class="usa-section padding-top-0 p0-bottom contact">
         <h1 class="font-sans-2xl line-height-46px margin-bottom-3">Contact</h1>
+        <div class="margin-top-2 margin-bottom-2">
+          <%= render partial: "practices/form/practice_last_updated", locals: {small_text: true} %>
+        </div>
```

Comment:
We can use the class margin-y-2 here, since both the top and bottom margin sizes are identical.